gdrive/settings.py

```diff
@@ -33,8 +34,8 @@
 ES_HOST = os.getenv("ES_HOST")
 ES_PORT = os.getenv("ES_PORT")
 
-QUALTRICS_APP_URL = os.getenv("QUALTRICS_APP_URL")
```

Comment:
Consistency with the other settings. Other modules use "host" for this instead of URL. Can change it back if this is inappropriate but its largely just for devs so I wasnt worried about it

Reply:
Well if you want to change it, you have to change it in manifest also or the application will not run

Reply:
I more confused now. Please make sure the settings file matches the manifest.